### ISSUE TITLE
False positive with comma-spacing and template strings.

### ISSUE THREAD

**Issue by doctyper** [eslint/eslint#1977]
Using:

```shell
$ eslint -v
v0.16.1
```

With the following `.eslintrc`:

```json
{
    "rules": {
        "comma-spacing": [2, {
            "before": false,
            "after": true
        }]
    },
    "env": {
        "es6": true
    }
}
```

The following throws an error:

```js
var hello = "Hello";
var string = `${hello}, world`; // Eslint: A space is required after ','  comma-spacing
```

Let me know if you need more information.

**Comment by doctyper** [comment]
My mistake, this bug only occurs with the `babel-eslint` parser. I'll open a ticket there.
